eclipse/sw360
# fix(ui): Fixed lar file to add missing widgets(Oauth Client & License Types)

Signed-off-by: Smruti Prakash Sahoo <smruti.sahoo@siemens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

Added missing widgets(Oauth Client and License Types) in Admin section.

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?

> How should these changes be tested by the reviewer?

In a fresh liferay install, upload the lar file and check if the Oauth client and License Types are visible and there is no error.

> Have you implemented any additional tests? - No

### Checklist
Must:
